public/src/js/widgets/columns/fronts.js

```diff
@@ -290,14 +290,12 @@ export default class Front extends ColumnWidget {
         }
 
         var collectionCap  = this.baseModel.state().defaults.collectionCap;
-        var articleNumber = this.collections().reduce((numberOfArticles, collection) => {
-            return numberOfArticles + collection.groups.reduce((numberOfArticles, group) => {
-                return numberOfArticles + group.items().length;
-            }, 0);
+        var articleNumber = item.group.parent.groups.reduce((numberOfArticles, group) => {
```

Comment:
the long accessor is quite scary `item.group.parent.groups`, it's safer to use `deepGet`.

Can you also check that this function is not called when you drag a sublink?